tests/figures/chart/chart_plugin.test.ts

```diff
@@ -44,7 +44,7 @@ jest.mock("../../../src/helpers/uuid", () => require("../../__mocks__/uuid"));
 
 let model: Model;
 
-function getChartConfiguration(model: Model, chartId: UID) {
+export function getChartConfiguration(model: Model, chartId: UID) {
```

Comment:
Try to avoid importing from a test file to another. That's an easy way to have mocked things and random beforeEach running and tests crashing for very hard to spot reasons 🙃 I think I created a chart_helper.ts file in some of my recent charts (unmerged) PRs